bears/general/InvalidLinkBear.py

```diff
@@ -26,8 +29,16 @@ def get_status_code_or_error(url, timeout):
         except requests.exceptions.RequestException:
             pass
 
-    @staticmethod
-    def find_links_in_file(file, timeout):
+    def get_status_code_checked(self, url, timeout):
```

Comment:
the point of this function is to raise a error on the second try.
the point of `get_status_code` is to return the code.

Is that okay?